src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  height: 40px;
+  width: 40px;
+  margin: 10px 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  justify-content: flex-start;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  display: flex;
+  text-decoration: none;
+  font-size: 12px;
+  margin-left: 20px;
+  color: #000;
+  font-weight: 500;
+  line-height: 60px;
+  font-style: normal;
+  position: relative;
+  text-transform: uppercase;
+}
+
+.nav__link:hover {
+  color: var(--main-color);
```

Comment:
Good job on using CSS variables for your main color. This makes your code more maintainable and easier to change in the future.